### DISCUSSION TITLE
Waiting for Github Copilot's access

### DISCUSSION THREAD

**Discussion by ozan1706** [community/community#8297] [Copilot Conversations]
I haven't had Github Copilot's access about a month. What should I do?

**Comment by TarunavBA** [comment]
To be honest, you have to wait.





~ 🤖

**Comment by jakobroenn02** [comment]
I have been waiting for 5 months. so good luck getting in.

**Comment by holasoymalva** [comment]
As GitHub Copilot requires state-of-the-art AI hardware, the maintainers are offering it to a limited number of testers for free, during the technical preview. They want to use the preview to learn how people use GitHub Copilot and what it takes to operate it at scale. If the technical preview is successful, their plan is to build a commercial version of GitHub Copilot in the future. To learn more about GitHub Copilot and their latest updates, please head over to 👉  https://copilot.github.com/ and wait, at the moment waiting is the only way to have access to the tool officially. My best wishes 👋 

BTW You can confirm your status by visiting 👉  https://github.com/features/copilot/signup
